tox.ini

```diff
@@ -143,7 +143,9 @@ commands=
     python -m pytest tests/pytest tests/cover/test_testdecorators.py
 
 [testenv:docs]
-deps = sphinx
+deps =
+    sphinx
+    django
 commands=sphinx-build -W -b html -d docs/_build/doctrees   docs docs/_build/html
```

Comment:
What is this section even for? AFAICT we don't use it anywhere.

If we do need it for something, should it not depend on a requirements file rather than specific deps?

Reply:
We don't use the doctest tox env - shall I just delete it?  As-is, I'd left it in in case someone was using it locally due to eg. the makefile not really working on windows.

Reply:
I think probably deleting it is best.